src/backend/integration/src/tests/list_proposal_reviews.spec.ts

```diff
@@ -0,0 +1,383 @@
+import { AnonymousIdentity, Identity } from '@dfinity/agent';
+import {
+  Actor,
+  PocketIc,
+  SubnetStateType,
+  generateRandomIdentity,
+} from '@hadronous/pic';
+import { describe, beforeAll, afterAll, it, expect } from 'bun:test';
+import {
+  Governance,
+  VALID_COMMIT_SHA_A,
+  VALID_COMMIT_SHA_B,
+  controllerIdentity,
+  createProposal,
+  createProposalReview,
+  createProposalReviewCommit,
+  createReviewer,
+  extractErrResponse,
+  extractOkResponse,
+  publishProposalReview,
+  setupBackendCanister,
+  validateProposalReview,
+} from '../support';
+import { _SERVICE } from '@cg/backend';
+import { Principal } from '@dfinity/principal';
+import { resolve } from 'path';
+
+const NNS_SUBNET_ID =
+  '2o3zy-oo4hc-r3mtq-ylrpf-g6qge-qmuzn-2bsuv-d3yhd-e4qjc-6ff2b-6ae';
+
+const NNS_STATE_PATH = resolve(
+  __dirname,
+  '..',
+  '..',
+  'state',
+  'proposal_reviews_nns_state',
+  'node-100',
+  'state',
+);
+
+describe('list proposal reviews', () => {
+  let actor: Actor<_SERVICE>;
+  let pic: PocketIc;
+  // set to any date after the NNS state has been generated
+  const initialDate = new Date(2024, 3, 25, 0, 0, 0, 0);
+
+  let governance: Governance;
+
+  let alice: Identity;
+  let aliceId: string;
+  let bob: Identity;
+  let bobId: string;
+
+  let proposal1Id: string;
+  let proposal2Id: string;
+
+  beforeAll(async () => {
+    pic = await PocketIc.create(process.env.PIC_URL, {
+      nns: {
+        state: {
+          type: SubnetStateType.FromPath,
+          path: NNS_STATE_PATH,
+          subnetId: Principal.fromText(NNS_SUBNET_ID),
+        },
+      },
+    });
+    await pic.setTime(initialDate.getTime());
+
+    const fixture = await setupBackendCanister(pic);
+    actor = fixture.actor;
+
+    governance = new Governance(pic);
+  });
+
+  afterAll(async () => {
+    await pic.tearDown();
+  });
+
+  beforeAll(async () => {
```

Comment:
Similarly, here, the setup is only done once. This one in particular helped a lot because there's a lot of setup here.